modules/print.py

```
@@ -0,0 +1,49 @@
+
+        return AkariDataSet()
+
+    def stream_call(self, data: AkariData, params: Any, callback: AkariModuleType | None = None) -> AkariDataSet:
+        self._logger.debug("PrintModule stream_call called")
+        self._logger.debug("Data: %s", data)
+        self._logger.debug("Params: %s", params)
+
+        last = data.last()
+        try:
+            self._logger.debug("Last Data: %s", json.dumps(last, indent=4))
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)